yfinance_cache/yfc_dat.py

```diff
@@ -90,11 +90,14 @@ class Interval(Enum):
 # USA:
 exchangeToXcalExchange["NYQ"] = "XNYS"
 exchangeToXcalExchange["ASE"] = exchangeToXcalExchange["NYQ"]
-exchangeToXcalExchange["PCX"] = exchangeToXcalExchange["NYQ"]  # NYSE Arca
```

Comment:
Are these comments necessary?

Reply:
the PCX mapping is necessary for my reproduction script to work

Reply:
feel free to delete them